### DISCUSSION TITLE
Windows 7 flyouts - background color

### DISCUSSION THREAD

**Discussion by Quisquose** [valinet/ExplorerPatcher#2946] [Q&A] [answered]
I love the functionality and layout of the Windows 7 flyouts (Clock, Volume Mixer etc.), but the white backgrounds are quite jarring on a dark theme.

I just wondered if there is any way to get the Windows 7 flyouts to have a darker background that better fits with dark mode?

![Win7 Clock](https://github.com/valinet/ExplorerPatcher/assets/7541455/74562ccb-ad82-42af-a51e-14c3b233765a)

![Win7 Sound](https://github.com/valinet/ExplorerPatcher/assets/7541455/cf4acb0a-2511-4d36-9c6b-8baada2f8e68)

**Comment by pyrates999** [comment]
That was before dark mode was a feature in windows, so probably not.

**Comment by OrthodoxWindows** [comment]
It is possible to achieve a true dark theme using a third-party visual style.

**Comment by pyrates999** [comment]
If you can find the way to do that, please post it then.

**Reply by sisericm** [reply, marked as answer]
Using a third party theme and patching through ultrauxtheme can solve this. However, be aware that how well the win 7 flyouts take to dark mode will depend on the particular theme you are using.
  
For example:
![Screenshot 2024-03-09 190920](https://github.com/valinet/ExplorerPatcher/assets/8255994/86ea9a6c-5ae8-4a39-9e44-40c3a7aa37d7)
My current theme does the mixer fine but the clock is a fail.
![Screenshot 2024-03-09 190904_cr](https://github.com/valinet/ExplorerPatcher/assets/8255994/d2aa5125-cbce-473e-9e15-5a2ac6db1252)
That said, I have used themes that will happily deal with both.

Useful links:
Ultrauxthemepatcher: [https://mhoefs.eu/software_uxtheme.php?ref=syssel&lang=en](url)
Superb themes: [https://www.deviantart.com/niivu](url)

**Reply by Quisquose** [reply]
Thank you, sisericm
